apis/datadoghq/v1alpha1/datadogpodautoscaler_types.go

```diff
@@ -227,32 +240,81 @@ type DatadogPodAutoscalerScalingRule struct {
 	Match *DatadogPodAutoscalerScalingRuleMatch `json:"match,omitempty"`
 }
 
-// DatadogPodAutoscalerUpdateMode defines the mode of the update policy.
-// +kubebuilder:validation:Enum:=ContainerMetrics
-type DatadogPodAutoscalerRecommenderName string
+// DatadogPodAutoscalerTargetType defines the type of the target.
+// +kubebuilder:validation:Enum:=Resource;ContainerResource
```

Comment:
is it explicit enough or should we explicitely say `PodResource` and `ContainerResource`?

Reply:
I have the same hesitation, changed back and forth, one is closer to upstream (`Resource`) and the other one more explicit `PodResource`).